### ISSUE TITLE
"Install another version" option is missing for GOG games

### ISSUE THREAD

**Issue by Elendil211** [lutris/lutris#3874]
Games that are installed from the source "GOG" don't have the option "install another version".

1. Start Lutris
2. Install game from GOG
3. Right click the game
4. Try to find the "install another version" option

**Comment by danieljohnson2** [comment]
I've taken a look at this, but I wonder what exactly this is supposed to do.

I enabled the command and I tried installing two versions of Final Doom off GOG; the result was two seemingly identical Final DOOM icons, but one was DosBOX and the other ZDoom. They were installed in the same directory, but this caused no problem for Final DOOM. I have to think that sharing a directory could be problematic for other games, though.

When I removed the games, I had to remove them separately, and this left me we an extra (grayed out) icon. I had to remove one of the icons again.

What's supposed to be happening here?

**Comment by Elendil211** [comment]
You can select a different directories, so you have two different versions of the same game. I often use this when I'm searching for bugs, so I can have two versions of the game.

**Comment by danieljohnson2** [comment]
Doesn't that leave you two identical looking icons? I suppose you can manually rename them, but better defaults would help here.

**Comment by Elendil211** [comment]
Yes, the icons look identical. I differentiate with something like "MyGame" and "MyGame_Test", etc.

**Comment by danieljohnson2** [comment]
I think I've figured out out to work this, though I'm going deeper into the guts of Lutris than my previous PRs did.

The tricky bits are ensuring that the new 'other version' installation wind up with distinct names and target directory paths. I've got it adding numerical suffixes when needed. These things are just defaults, so it shouldn't need to be real smart.

Also, I had issues with installing using a stale Game object that the game-bar was keeping. I've added code to re-fetch this so it can reliably tell when it is installed already.
